ajax/libs/restful.js/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "restful.js",
+  "filename": "restful.min.js",
+  "version": "0.9.6",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/marmelab/restful.js"
+  },
+  "description": "A pure JS client for interacting with server-side RESTful resources. Think Restangular without Angular.",
+  "author": "Robin Bressan <robin@buddey.net>",
+  "license": "MIT",
+  "dependencies": {
+    "immutable": "3.7.6",
+    "object-assign": "2.0.0",
+    "qs": "5.2.0",
+    "warning": "2.1.0"
+  },
+  "homepage": "https://marmelab.com/blog/2015/03/10/deal-easily-with-your-rest-api-using-restful-js.html",
+  "keywords": [
+    "rest",
+    "restful",
+    "api"
+  ],
+  "npmName": "restful.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "!(es5).*"
```

Comment:
I think it's not flexible because It will not grab files of directory.
I suggest to use this:
```json
"files": [
  "！（es5)/*",
  "restful*"
]
```
How do you think? @nolsherry